dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -218,6 +218,14 @@ dependent_startup_wait_for=rsyslogd:running {% if delay_non_critical_daemon %}de
     {% if thermalctld.thermal_monitor_update_elapsed_threshold is defined and thermalctld.thermal_monitor_update_elapsed_threshold is not none %}
         {%- set options = options + " --thermal-monitor-update-elapsed-threshold " + thermalctld.thermal_monitor_update_elapsed_threshold|string %}
     {% endif -%}
+
+    {% if enable_liquid_cooling %}
```

Comment:
@yuazhe could you follow this approach (as in this PR : https://github.com/sonic-net/sonic-buildimage/pull/23139#issue-3185807611) - to define the configs under "thermalctld" ? 
e.g. below
{
    "thermalctld": {
        "enable_liquid_cooling": True,
        "liquid_cooling_update_interval ": 5
    }
}

Reply:
you would need to access using below ?
thermalctld.enable_liquid_cooling
thermelctld.liquid_cooling_update_interval